tests/index.js

```diff
@@ -0,0 +1,21 @@
+const tape = require('tape')
+const params = require('../index.js')
+
+tape('JSON validity', function (t) {
```

Comment:
Attempting to load `index.js` should be enough to tell whether the JSON in the three files is valid or not.  If it isn't, there will be an exception thrown.